desc/coils.py

```diff
@@ -2331,15 +2407,17 @@ def _compute_position(self, params=None, grid=None, **kwargs):
             "grid must be supplied to MixedCoilSet._compute_position, since the "
             + "default grid for each coil could have a different number of nodes.",
         )
+        kwargs.setdefault("basis", "xyz")
         params = self._make_arraylike(params)
         grid = self._make_arraylike(grid)
-        x = jnp.vstack(
-            [
-                coil._compute_position(par, grd, **kwargs)
-                for coil, par, grd in zip(self.coils, params, grid)
-            ]
-        )
-        return x
+        out = []
+        for coil, par, grd in zip(self.coils, params, grid):
+            out.append(coil._compute_position(par, grd, dx1, **kwargs))
```

Comment:
Check that this is correct, because I think the existing test should cover the logic of `sym=True` and `dx1=True` for `CoilSet._compute_position` (see previous comment).